static/new_home/style.css

```diff
@@ -523,6 +522,12 @@ header
   }
 }
 
+@media screen and (max-width:600px) {
+  img {
+    max-width: 275px;
```

Comment:
With this size too, some part of the images and the text below cuts out on small screens

Reply:
Should I make changes now as this is merged?

Reply:
No, I've made the changes myself before merging it.